### PR TITLE
ci: Update java to 17

### PR DESCRIPTION
**Pull request by Dor-bl**
due to failure when trying to update to the latest gradle version, we need to update Java version to 17 first thing 
* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin 'com.android.internal.application'.
   > Android Gradle plugin requires Java 17 to run. You are currently using Java 11.
      Your current JDK is located in /usr/lib/jvm/temurin-11-jdk-amd64
      You can try some of the following options:
       - changing the IDE settings.
       - changing the JAVA_HOME environment variable.
       - changing `org.gradle.java.home` in `gradle.properties`.

### CODE REVIEW

**Review by KazuCocoa — APPROVED** [review]
its fine. This repository is for testing app purpose only. We may need to keep minSdkVersion as lower one (but I guess we could bump it up to uia2's minSdkVersion for now) but others are basically ok to update

**Comment by mykola-mokhnach** [comment]
We probably need to mention in readme about this requirement

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by Dor-bl** [comment]
> We probably need to mention in readme about this requirement

Updated README.md

File: README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ android-apidemos
 
 A fork of Google's Android ApiDemos application, used for testing Appium
 
+#### Prerequisites
```
Comment (mykola-mokhnach) [inline README.md:6]: Requirements

File: README.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ android-apidemos
...
+
+- Java 17
```
Comment (mykola-mokhnach) [inline README.md:8]: npm/node are also needed to run the build

**Review by mykola-mokhnach — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Dor-bl) [reply README.md:8]: done

**Pull request merged**